### PR TITLE
Ian contests

### PR DESCRIPTION
**Pull request by LlanddewiLovesYou**
Yeah so, I wrote this code so you should probably look at it because, yknow... it will need it.

### CODE REVIEW

File: src/components/app.jsx [hunk 1]
```diff
@@ -11,7 +11,8 @@ import "./App.css";
 
 class App extends Component {
   static propTypes = {
-    authenticateUser: PropTypes.func.isRequired
+    authenticateUser: PropTypes.func.isRequired,
+    fetchCurrentUser: PropTypes.func.isRequired
```
Comment (Bhammy) [inline src/components/app.jsx:16]: Is this supposed to be fetchContest?

File: src/index.js [hunk 2]
```diff
@@ -14,4 +15,6 @@ ReactDOM.render(
   document.getElementById("root")
 );
 
+window.fetchContest = fetchContest
```
Comment (Bhammy) [inline src/index.js]: is this for testing purposes? otherwise we don't need this on the window in the production version, I think.

**Review by Bhammy — CHANGES_REQUESTED** [review]
Looks good to me! Two minor questions/comments.

**Comment by LlanddewiLovesYou** [comment]
Wha? And yes.

**Comment by Bhammy** [comment]
Not sure which of your responses goes to which of my comments - if it's in order, I'm wondering if fetchCurrentUser is supposed to be fetchContest because authenticateUser already calls receiveCurrentUser on the backend. For the other, yeah, just remove the window.fetchContest for the version to merge with the master branch.

**Comment by LlanddewiLovesYou** [comment]
It was in order, yes. I don't know. I didn't write anything having to do
with users. That's some weird merge stuff

**Comment by Bhammy** [comment]
Okay, that's fine. Just make the changes locally (delete the window.fetchContest and the fetchCurrentUser) and commit again and I can approve. Thanks! :)

**Comment by matthaws** [comment]
So the fetchCurrentUser thing is not a mistake, its on the App component to get the current user data if it detects the auth token stored locally. Check out our recent other additions!

**Review by Bhammy — APPROVED** [review]

**Pull request merged**